apps/meteor/client/views/room/RoomOpener.tsx

```diff
@@ -60,6 +62,10 @@ const RoomOpener = ({ type, reference }: RoomOpenerProps): ReactElement => {
 							return <RoomNotFound />;
 						}
 
+						if (error instanceof NotSubscribedToRoomError && error.details) {
+							return <NotSubscribedRoom rid={error.details.rid} roomType={error.details.roomType} />;
```

Comment:
```suggestion
							return <NotSubscribedRoom rid={error.details.rid} roomType={type} />;
```

Reply:
You don't need to add type to the error, as `type` is already present in this components closure (see the props)